### ISSUE TITLE
installation impossible

### ISSUE THREAD

**Issue by amo13** [pagekit/pagekit#947]
## Problem

After unpacking the download to my root folder for my nginx virtual host and reloading nginx with the additional virtual host file, I get the following error in my browser:

```
ContextErrorException in DocParser.php line 967: Notice: Trying to access array offset on value of type null

    in DocParser.php line 967
    at ErrorHandler->handleError() in DocParser.php line 967
    at DocParser->Identifier() in DocParser.php line 665
    at DocParser->Annotation() in DocParser.php line 641
    at DocParser->Annotations() in DocParser.php line 334
    at DocParser->parse() in DocParser.php line 496
    at DocParser->collectAnnotationMetadata() in DocParser.php line 718
    at DocParser->Annotation() in DocParser.php line 641
    at DocParser->Annotations() in DocParser.php line 334
    at DocParser->parse() in SimpleAnnotationReader.php line 75
    at SimpleAnnotationReader->getMethodAnnotations() in AnnotationLoader.php line 64
    at AnnotationLoader->load() in RoutesLoader.php line 91
    at RoutesLoader->addController() in RoutesLoader.php line 52
    at RoutesLoader->load() in Router.php line 167
    at Router->getRouteCollection() in Router.php line 187
    at Router->getMatcher() in Router.php line 269
    at Router->match() in RouterListener.php line 47
    at RouterListener->onRequest()
    at call_user_func_array() in EventDispatcher.php line 123
    at EventDispatcher->trigger() in HttpKernel.php line 72
    at HttpKernel->handle() in Application.php line 67
    at Application->run() in app.php line 29
    at require_once('/srv/pagekit/app/installer/app.php') in index.php line 42
```

I checked the server requirements and I do have all necessary php extensions loaded.

## Technical Details

- Pagekit version: 1.0.18
- Webserver: nginx
- Database: sqlite
- PHP Version: 7.4

**Comment by manuelguzman** [comment]
This is due to a PHP7.4 issue in doctrine/annotator. The installation composer.json requires "doctrine/annotations": "~1.2.3"
But the bug in annotator is not fixed until 1.7.0. 

To resolve yourself update composer.json to require
"doctrine/annotations": "~1.7.0"

Then update composer for that package:
composer.phar update doctrine/annotations

**Comment by amo13** [comment]
Thank you!
